swank/src/test/scala/org/ensime/server/protocol/swank/SwankFormatsSpec.scala

```diff
@@ -58,24 +57,24 @@ class SwankFormatsSpec extends FlatSpec with Matchers with EnsimeTestData {
     )
 
     unmarshal(
-      s"""(swank:import-suggestions $file1_str 1 ("foo" "bar") 10)""",
+      s"""(swank:import-suggestions "$file1" 1 ("foo" "bar") 10)""",
       ImportSuggestionsReq(file1, 1, List("foo", "bar"), 10): RpcRequest
     )
   }
 
   it should "unmarshal RpcAnalyserRequests" in {
     unmarshal(
-      s"""(swank:remove-file $file1_str)""",
+      s"""(swank:remove-file "$file1")""",
```

Comment:
The "unmarshal" tests fail: $file1 etc  contain unescaped backslashes that break string parsing